packages/tupaia-web/src/api/queries/useReport.ts

```diff
@@ -20,7 +20,16 @@ type QueryParams = {
 };
 
 export const useReport = (reportCode: DashboardItemType['reportCode'], params: QueryParams) => {
-  const { dashboardCode, projectCode, entityCode, itemCode, startDate, endDate, legacy } = params;
+  const {
+    dashboardCode,
+    projectCode,
+    entityCode,
+    itemCode,
+    startDate,
+    endDate,
+    legacy,
+    ...rest
```

Comment:
So rest is just always `reportDrillDownId` right? Assuming so, can we explicitly add that to the list?

Reply:
In the URL it is `reportDrilldownId` but in reality when we use `parameterLink` from the parent it could be any property name, e.g. `eventId` etc

Reply:
Oh now I see what's going on. That's the key field from the parent data. Ok please ignore my question.